startup.sh

```diff
@@ -1,12 +1,4 @@
 #!/bin/sh
-
-git -C . fetch || git clone https://github.com/vladmandic/automatic.git .
-
-if [ ! -d /deps/venv ]; then
-    echo "Creating venv with system packages (torch, ipex, etc)"
-    python -m venv /deps/venv --system-site-packages
-fi
-
-export PATH=/deps/venv/bin:$PATH
-
-python ./launch.py "$@"
+git -C automatic fetch || git clone https://github.com/vladmandic/automatic.git
+cd automatic
```

Comment:
It's better to keep the original directory structure to keep maximal backward compatibility.